SecurityAgent/SecurityAgent.h

```diff
@@ -22,9 +22,10 @@
 #include "Module.h"
 #include "AccessControlList.h"
 
-#include <securityagent/IPCSecurityToken.h>
 #include <interfaces/json/JsonData_SecurityAgent.h>
 
+#include <core/FileObserver.h>
```

Comment:
Thats a bit strange, this file should be loaded by the <core/core.h> in Module.H. I guess it is not needed.

Reply:
done